js/util/helpers.js

```diff
@@ -214,38 +219,247 @@ define(["dao", "globals", "lib/knockout", "util/eventLog"], function (dao, g, ko
     function getTeamsDefault() {
         var teams;
 
-        teams = [
```

Comment:
I'm fine with most of your formatting changes, but can you revert this one? Makes the list of teams unwieldy to look at.